Pass --threads=0 to xz so that it will use as many threads as there are cores to compress the packages.  This makes qemu's do_package_ipk run in 34s instead of 275s on my machine.

Upstream-Status: Inappropriate (ugly but works, discussing with upstream now)
Signed-off-by: Ross Burton <ross.burton@intel.com>

diff --git a/opkg-build b/opkg-build
index 0050c2b..93a7956 100755
--- a/opkg-build
+++ b/opkg-build
@@ -230,6 +230,8 @@ if gzip --help 2>&1 | grep -- "-T" > /dev/null; then
 fi
 if [ $compressor = "gzip" ] ; then
 	compressorargs=$zipargs
+elif [ $compressor = "xz" ] ; then
+	compressorargs="--threads 0"
 fi
 
 tsortargs=
